web/tests/libtest/codechecker.py

```diff
@@ -582,6 +582,9 @@ def wait_for_server_start(stdoutfile):
                 out = f.read()
                 if "Server waiting for client requests" in out:
                     return
+                if "usage: CodeChecker" in out:
```

Comment:
Could you add a comment about this is an error case when the server failed to start and gave some error message.